screens/TabTwoScreen.tsx

```diff
@@ -1,32 +1,41 @@
-import * as React from 'react';
-import { StyleSheet } from 'react-native';
+import React, { useState, useEffect } from 'react';
```

Comment:
Could you create this ImagePickerExample as a separate screen and separate file (which you can add to the tab bar)? There's some cool stuff on TabTwoScreen now on develop branch like a counter, so I don't want to completely replace this.